zebra-consensus/src/block/check.rs

```diff
@@ -58,24 +58,22 @@ pub fn coinbase_is_first(block: &Block) -> Result<Arc<transaction::Transaction>,
     Ok(first.clone())
 }
 
-/// Returns `Ok(())` if `hash` passes:
-///   - the target difficulty limit for `network` (PoWLimit), and
-///   - the difficulty filter,
-/// based on the fields in `header`.
+/// Returns `Ok(ExpandedDifficulty)` if `difficulty_threshold` of `header` passes
```

Comment:
Nitpick: Now there's only one check, we can be a bit more specific here.

```suggestion
/// Returns `Ok(ExpandedDifficulty)` if the`difficulty_threshold` of `header` is at least as difficult as
```